### PR TITLE
Add iiNet

### PR DESCRIPTION
**Pull request by fletcherdodd**

### CODE REVIEW

**Comment by Carlgo11** [comment]
Our guidelines on [new site additions](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#guidelines) require new entries to have a global rank of 200,000 or less on [Similarweb](https://www.similarweb.com/website/).
This requirement is in place because maintaining the lists takes time, and a list of all websites on the internet isn't feasible.

Unfortunately, as the site you want to add is ranked above 200,000, it doesn't meet our guidelines and thus can't be added. If the site rises in popularity in the future and is ranked below 200,000 on Similarweb, this pull request can be reopened.

To prevent any of your future pull requests from being closed, please make sure your contribution adheres to our [CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md) and [EXCLUSION.md](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md) before creating a new pull request. Thank you!

**Comment by fletcherdodd** [comment]
Same reply for #8112 & #8106 applies to this one, [see here](https://www.similarweb.com/website/iinet.net.au).

File: entries/i/iinet.net.au.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "iiNet": {
+    "domain": "iinet.net.au",
+    "additional-domains": [
+      "iinet.com.au"
+    ],
+    "tfa": [
+      "sms"
+    ],
```
Comment (hkamran80) [inline entries/i/iinet.net.au.json:9]: This paragraph of the documentation seems more like [authorization than authentication](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#authorization) to me.

> At iiNet, multi-factor authentication is used when a customer contacts us to access or change personal or account information. This is to help ensure someone isn’t trying to access your information or make changes to your account without your permission.

Comment (hkamran80) [reply entries/i/iinet.net.au.json:9]: Never mind. I misread it. The "Toolbox" terminology threw me off.

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**